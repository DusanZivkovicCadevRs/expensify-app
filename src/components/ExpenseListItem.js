import React from 'react';
import { Link } from 'react-router-dom';

// Export a stateless functional component
// description, amount, createdAt

const ExpenseListItem = ({ id, description, amount, createdAt }) => (
    <div>
        <Link to={`/edit/${id}`} >
            <h1>{description}</h1>
        </Link>
        <p>{amount} - {createdAt}</p>

    </div>
);

export default ExpenseListItem;
