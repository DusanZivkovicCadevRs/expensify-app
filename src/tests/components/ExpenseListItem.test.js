import React from 'react';
import { shallow } from 'enzyme';
import expenseFixtures from '../fixtures/expenses';
import ExpenseListItem from '../../components/ExpenseListItem';

test('should render ExpenseListItem', () => {
    const wrapper = shallow(
        <ExpenseListItem
            {...expenseFixtures[0]}
        />
    );
    expect(wrapper).toMatchSnapshot();
});
