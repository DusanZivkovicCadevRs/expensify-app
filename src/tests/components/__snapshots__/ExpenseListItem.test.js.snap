// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem 1`] = `
<div>
  <Link
    replace={false}
    to="/edit/0"
  >
    <h1>
      Bill
    </h1>
  </Link>
  <p>
    169
     - 
    0
  </p>
</div>
`;
